aconfig_test.go

```diff
@@ -420,6 +420,49 @@ func TestFlag(t *testing.T) {
 	}
 }
 
+func TestFlagAllowUnknownFlags(t *testing.T) {
+	var cfg struct {
+		A string `flag:"a" env:"a" usage:"a" required:"true"`
+		B string `flag:"b" env:"b" usage:"b" required:"true"`
+	}
+
+	loader := LoaderFor(&cfg, Config{
+		SkipDefaults:      true,
+		SkipFiles:         true,
+		SkipEnv:           true,
+		SkipFlags:         false,
+		AllowUnknownFlags: true,
+		EnvPrefix:         "",
+		FlagPrefix:        "",
+		Files:             []string{},
+		FileDecoders:      map[string]FileDecoder{},
+		Args: []string{
+			"--a",
+			"foo",
+			"--b",
+			"bar",
+			"--c",
+			"foobar",
```

Comment:
This actually doesn't work, if we add `--d foobar` in first and try it - it will fail to parse `--a` and `--b`...